lib/global.gi

```diff
@@ -147,13 +147,31 @@ InstallGlobalFunction( IsReadOnlyGlobal,
     return isro;
 end);
 
+#############################################################################
+##
+#F  IsConstantGlobal ( <name> ) . determine if a global variable is constant
+##
+##  IsConstantGlobal ( <name> ) returns true if the global variable
+##  named by the string <name> is constant and false otherwise (the default)
+##
+
+InstallGlobalFunction( IsConstantGlobal,
+        function (name)
+    local isro;
+    CheckGlobalName( name );
+    isro := IS_CONSTANT_GLOBAL(name);
```

Comment:
Why "isro" ? Probably copy & paste from `IsReadOnlyGlobal`, but this name doesn't make sense here.